File: c/1019_time_conversion.c
/*  
    @autor: Guiventura6;
    @data: 21/01/2022;
    @Problema: Time Conversion;    
*/

#include <stdio.h>

int main(){
    
    int duration, hours, minutes, seconds, rest;

    scanf("%d", &duration);

    hours = duration / 3600;
    rest = duration % 3600;
    minutes = rest / 60;
    seconds = rest % 60;
        
    printf("%d:%d:%d\n", hours, minutes, seconds);
    
    return 0;
}

File: python/1019_time_conversion.py
'''
    Autor: Guiventura6
    Data: 03/03/2022
    Modified: (29/06/2023,)
    Problema: Time Conversion
    Resume: Read an integer value, which is the duration in seconds of acertain
    event in a factory, and inform it expressed in hours:minutes:seconds.
'''
duration = int(input())

m, s = divmod(duration, 60)
h, m = divmod(m, 60)

print('{:d}:{:d}:{:d}'.format(h, m, s))
